cciskel-duffy

```diff
@@ -21,7 +22,7 @@ import argparse
 
 url_base="http://admin.ci.centos.org:8080"
 api_key=open(os.path.expanduser('~/duffy.key')).read().strip()
-cached_state_path = os.path.expanduser('~/duffy-state-cache.json')
+cached_state_path = os.path.expanduser('~/.duffy-state-cache.json')
```

Comment:
@cgwalters this sounds fine to me. I only moved it there because it felt very much like it should be out of the way. I'm good with a /path/to/jenkins-state structure.